project/plugins.sbt

```diff
@@ -1,4 +1,6 @@
-addSbtPlugin("org.spark-packages" % "sbt-spark-package" % "0.2.5")
+resolvers += "bintray-spark-packages" at "https://dl.bintray.com/spark-packages/maven/"
```

Comment:
We need to push resolver here for the sbt-spark-package plugin because the directory layout is different.